code/lib/cli/src/automigrate/fixes/remove-jest-testing-library.test.ts

```diff
@@ -56,9 +56,9 @@ it('should prompt to install the test package and run the codemod', async () =>
     - @storybook/jest
     - @storybook/testing-library
 
-    Install the replacement for those packages: @storybook/test
+    We will uninstall them for you and install @storybook/test instead.
 
-    And run the following codemod:
+    Also, we will apply the following codemod to your stories to automatically migrate them to the new package:
      npx storybook migrate migrate-to-test-package --glob="**/*.stories.@(js|jsx|ts|tsx)"
```

Comment:
This seems like irrelevant detail. If we're doing the codemod automatically there's no reason to tell the user the exact command we're running, that's just an implementation detail.

_If_ it's because we want the user to be able to skip this migration and do it manually later, we should just write that out explicitly. Eg.

> We'll migrate your usage of the packages to `@storybook/test`. If you prefer to do it manually later, you can run the following codemod: XYZ

I don't personally see the reason to call it out though.